### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GrantBirki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,12 +28,13 @@ A community made GraphQL API for Escape from Tarkov
 
 - View the prices of items
 - Get detailed ammo, armor, and weapon information
-- Fetch flea market data
+- Fetch real-time flea market data
```
Comment (thaddeus) [inline README.md:31]: This isn't entirely accurate, probably the original description is more so.

**Review by thaddeus — APPROVED** [review]
One potential change, but mostly a nit, overall LGTM 👍

File: README.md (hunk 1 above)
Comment (GrantBirki) [reply README.md]: Reverted

**Review by Razzmatazzz — APPROVED** [review]

**Comment by GrantBirki** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Pull request merged**